app/views/labs/show.html.erb

```diff
@@ -54,16 +54,16 @@
       <thead>
         <tr>
           <th>Espacio</th>
-          <th>Responsable</th>
-          <th>Horarios</th>
+          <th>Contacto</th>
+          <th>Horario</th>
           <th>Reservar</th>
         </tr>
       </thead>
       <tbody>
         <% @lab.lab_spaces.each do |lab_space| %>
           <tr class="LabSpace--listitem">
             <td><%= lab_space.name %></td>
-            <td>Dra. Alejandra S&aacute;nchez</td>
+            <td><%= lab_space.contact_email %></td>
```

Comment:
Thanks for this 😂😂

Reply:
We should move the contents of this loop to a partial so we can just do `<%= render @lab_spaces %>`